modules/serverless/main.tf

```
@@ -62,6 +62,14 @@ resource "aws_emrserverless_application" "this" {
     }
   }
 
+  dynamic "interactive_configuration" {
+    for_each = [var.interactive_configuration]
```

for this we have to check for a length, otherwise the default fallbacks of the `try()` will be used even when a value isn't provided:

```suggestion
    for_each = length(var.interactive_configuration) > 0 ? [var.interactive_configuration] : []
```

---

Ohhh, I see, thank you!